ext/nnstreamer/tensor_filter/tensor_filter_llamacpp.cc

```diff
@@ -0,0 +1,350 @@
+/**
+ * GStreamer Tensor_Filter, llamacpp Module
+ * Copyright (C) 2024 Yelin Jeong <yelini.jeong@samsung.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * @file    tensor_filter_llamacpp.cc
+ * @date    5 Dec 2024
+ * @brief   llamacpp module for tensor_filter gstreamer plugin
+ * @see     http://github.com/nnstreamer/nnstreamer
+ *          https://github.com/ggerganov/llama.cpp
+ * @author  Yelin Jeong <yelini.jeong@samsung.com>
+ * @bug     No known bugs except for NYI items
+ * @details Pipeline example
+ *          gst-launch-1.0 filesrc location=input.txt ! application/octet-stream !
+ *          tensor_converter ! other/tensors,format=flexible !
+ *          tensor_filter framework=llamacpp model=llama-2-7b-chat.Q2_K.gguf invoke-dynamic=TRUE !
+ *          other/tensors,format=flexible ! tensor_decoder mode=octet_stream !
+ *          application/octet-stream ! filesink location=output.txt
+ */
+
+#include <assert.h>
+#include <errno.h>
+#include <nnstreamer_cppplugin_api_filter.hh>
+#include <nnstreamer_log.h>
+#include <nnstreamer_plugin_api_util.h>
+#include <nnstreamer_util.h>
+#include <vector>
+
+#include "ggml.h"
+#include "llama.h"
+/**
+ * @brief Macro for debug mode.
+ */
+#ifndef DBG
+#define DBG FALSE
+#endif
```

Comment:
Thanks for the great PR. :+1: 
This file does not use DBG, how about removing this?

Reply:
Removed unused value!
Thank you :)